docs/settings.rst

```diff
@@ -1272,6 +1272,21 @@ Maximum number of milliseconds to queue a query. See :doc:`performance`.
 
 Number of receiver (listening) threads to start. See :doc:`performance`.
 
+.. _setting-resolver:
+
+``resolver``
+------------
+
+-  IP Address with port
```

Comment:
```suggestion
-  IP Address with optional port
```
It says "with port" and then one of the two examples is without a port, which is a bit confusing.

Reply:
thanks, applied